pyodide_build/mkpkg.py

```diff
@@ -138,7 +138,7 @@ def _download_wheel(pypi_metadata: URLDict) -> Iterator[Path]:
 
 
 def run_prettier(meta_path: str | Path) -> None:
-    subprocess.run(["npx", "prettier", "-w", meta_path])
+    subprocess.run(["npx", "prettier", "-w", meta_path], check=False)
```

Comment:
If we set this to `True`, we could avoid pre-commit.ci adding more commits to people's PRs.

```suggestion
    subprocess.run(["npx", "prettier", "-w", meta_path], check=True)
```